.gitignore

```diff
@@ -34,4 +34,7 @@ customs/uis/*
 .gulp-scss-cache
 node_modules
 storage
+bin/composer
```

Comment:
Thanks, it won't hurt, but we shouldn't be tracking (in this case untracking) those, the person installing should clean the directory after installation, and I believe the composer-setup clean all this.